### DISCUSSION TITLE
Multiple languages

### DISCUSSION THREAD

**Discussion by sviperm** [barrust/pyspellchecker#97] [Ideas]
How about idea to load multiple languages?

```python
from spellchecker import SpellChecker
russian = SpellChecker(language=['en', 'ru'])  # use the English and Russian languages?
```

Just load multiple dictionaries and combine them, if same word in dictionaries, then just sum values
```python
en_dict = {
    'hello': 137,
    'world': 150,
}
ru_dict = {
    'водка': 150,
    'hello': 120,
}
union_dict = {
    'hello': 257,
    'world': 150,
    'водка': 150,
}

**Comment by barrust** [comment]
This is a great idea. I should be able to put something like this into the next version without much of a lift to implement. Thanks for the idea!

**Comment by msalhab96** [comment]
It's a great idea, I think one thing that shall be taken into the consideration is the memory side as more vocabulary will be loaded to it, we might like load the most frequent ones in each lang, unless the user specify to load the whole data
